internal/cmd/skupper/token/kube/token_issue.go

```diff
@@ -96,18 +96,22 @@ func (cmd *CmdTokenIssue) ValidateInput(args []string) []error {
 
 	// Validate flags
 	//TBD is there a limit to number of redemptions
-	if cmd.Flags.RedemptionsAllowed < 1 {
+	if cmd.Flags != nil && cmd.Flags.RedemptionsAllowed < 1 {
 		validationErrors = append(validationErrors, fmt.Errorf("number of redemptions is not valid"))
 	}
 
-	//TBD what is valid times
-	if cmd.Flags.ExpirationWindow <= 0*time.Minute {
-		validationErrors = append(validationErrors, fmt.Errorf("expiration time is not valid"))
+	if cmd.Flags != nil && cmd.Flags.ExpirationWindow.String() != "" {
```

Comment:
If I specified a expiration window of less than 10 seconds this check is not going to pass (which is correct), but I wonder if it is correct to show that the minimum expiration window is 10 seconds, that is an amount of time too small to operate with. We might need an ExpirationWindowValidator (minimum: 5 minutes? 15 minutes?) in the same fashion as the TimeoutValidator, but it can be done in a future pull request.

Reply:
agree that makes sense

Reply:
#1729